src/components/MainContent/Content.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { ContentProps } from './Components';
+
+const SplitByNewLine = (text: string) => {
```

Comment:
Since this is a normal function and not a component, you should use standard camel-casing for the name.